### PR TITLE
Migrate to k8s.io/klog from glog.

### PR DESCRIPTION
**Pull request by humblec**
Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

### CODE REVIEW

File: Gopkg.toml [hunk 1]
```diff
@@ -45,6 +45,10 @@
   name = "k8s.io/csi-api"
   version = "kubernetes-1.13.2"
 
+[[constraint]]
+  name = "k8s.io/klog"
+  revision = "8139d8cb77af419532b33dfa7dd09fbc5f1d344f"
```
Comment (pohly) [inline Gopkg.toml]: See https://github.com/kubernetes-csi/external-provisioner/pull/219#discussion_r253469203

Comment (humblec) [reply Gopkg.toml]: @pohly I updated the commit message mentioning why `klog` has been updated as `constraint`.  Regarding `revision` , I was locking it to the same revision kubernetes has today https://github.com/kubernetes/kubernetes/blob/master/Godeps/Godeps.json#L4052 . Please let me know if any thing else has to be done/changed here.

Comment (pohly) [reply Gopkg.toml]: Conceptually, `Godeps.json` corresponds to `Gopkg.lock`. If the goal is to use the same model as in Kubernetes, then a) don't add a revision to `Gopkg.toml` and b) specify a certain revision that you want to update to with `dep ensure -update k8s.io/klog@<revision>`.

Having said that, I don't think that we absolutely need to use exactly the same klog revision as Kubernetes. Using the latest stable version is fine, and as klog doesn't have releases, that means following the master branch.

Comment (pohly) [reply Gopkg.toml]: Actually, k8s.io/klog does have releases. We should use those and just let dep pick the latest one.

File: cmd/csi-attacher/main.go [hunk 2]
```diff
@@ -67,30 +67,31 @@ var (
 
 func main() {
 	flag.Set("logtostderr", "true")
+	klog.InitFlags(nil)
```
Comment (pohly) [inline cmd/csi-attacher/main.go]: Hmm, does `flag.Set("logtostderr", "true")` work when it comes before the definition of the `logtostderr` flag itself in `klog.InitFlags(nil)`?

Comment (jsafrane) [reply cmd/csi-attacher/main.go]: It doesn't :-(

Comment (pohly) [reply cmd/csi-attacher/main.go]: No, it doesn't, I tried. `klog.InitFlags(nil) must come first.

Comment (jsafrane) [reply cmd/csi-attacher/main.go]: It's fixed now.

**Pull request merged**